src/Analyzers/Core/CodeFixes/MatchFolderAndNamespace/AbstractChangeNamespaceToMatchFolderCodeFixProvider.CustomFixAllProvider.cs

```diff
@@ -39,8 +39,9 @@ private class CustomFixAllProvider : FixAllProvider
                 if (diagnostics.IsDefaultOrEmpty)
                     return null;
 
+                var title = FixAllContextHelper.GetDefaultFixAllTitle(fixAllContext);
```

Comment:
why?

Reply:
> why?

Possibly I was using in as equivalence key and changed that in a later commit.